packages/dd-trace/src/config.js

```diff
@@ -324,9 +355,11 @@ ken|consumer_?(?:id|key|secret)|sign(?:ed|ature)?|auth(?:entication|orization)?)
       port: String(coalesce(dogstatsd.port, process.env.DD_DOGSTATSD_PORT, 8125))
     }
     this.runtimeMetrics = isTrue(DD_RUNTIME_METRICS_ENABLED)
+    this.tracePropagationStyle = {
+      inject: DD_TRACE_PROPAGATION_STYLE_INJECT,
+      extract: DD_TRACE_PROPAGATION_STYLE_EXTRACT
+    }
     this.experimental = {
-      b3: isTrue(DD_TRACE_B3_ENABLED),
```

Comment:
Let's continue to support these for now for backward compatibility, but only when the new option is not used.

Reply:
That's what the use of those flags as defaults in the propagation style configs does. That gives us one code path and a simpler bit of code to delete later when we're ready to deprecate the experimental flags.

Reply:
Ah yes I noticed that after posting this, but it still shouldn't be a single code path because they work differently and could be conflicting when making the switch.